src/ucp/core/ucp_ep.c

```diff
@@ -874,18 +874,27 @@ static void ucp_ep_close_flushed_callback(ucp_request_t *req)
                                       &req->send.disconnect.prog_id);
 }
 
-static ucs_status_t ucp_ep_close_nb_check_params(ucp_ep_h ep, unsigned mode)
+static int ucp_ep_close_is_param_flag(const ucp_request_param_t *param,
```

Comment:
better introduce common function, which would return flag if present and 0 otherwise? The it can be reused by other APIs as well

Reply:
also will not need to change `ucp_ep_close_nb_check_params`

Reply:
ok, fixed